app/views/home/index.html.erb

```diff
@@ -2,60 +2,62 @@
 
 
 
-<div class="feed-send">
+    <div class="feed-send">
 
-<%= form_for(@post, url: posts_url, method: 'post') do |f| %>
+        <%= form_for(@post, url: posts_url, method: 'post') do |f| %>
 
-<div class="field">
-<%= f.label :title, "Title" ,class: "form-letter"%> <br>
-<%= f.text_field :title , class: "ftext"%> <br>
-</div>
+        <div class="field">
+            <%= f.label :title, "Title" ,class: "form-letter"%> <br>
+            <%= f.text_field :title , class: "ftext"%> <br>
+        </div>
 
-<div class="field">
-<%= f.label :content, "Content" ,class: "form-letter" %> <br>
-<%= f.text_area :content , class: "ftext"%> <br>
-</div>
+        <div class="field">
+            <%= f.label :content, "Content" ,class: "form-letter" %> <br>
+            <%= f.text_area :content , class: "ftext"%> <br>
+        </div>
 
-<div class="field">
-<%= f.submit "Send" ,class: "form-letter" %>
-</div>
-<% end %>
-</div>
+        <div class="field">
+            <%= f.submit "Send" ,class: "form-letter" %>
+        </div>
+        <% end %>
+    </div>
 
 
-<div class="feed-posts">
-<% @posts.each do |post| %> 
+    <div class="feed-posts">
+        <% @posts.each do |post| %>
 
-<div class="post-field">
-    <%= post.content %>  <br>
-    <%= "By: " +  User.find(post.user_id).name %>  <br>
+        <div class="post-field">
+            <%= post.content %> <br>
+            <%= "By: " +  User.find(post.user_id).name %> <br>
```

Comment:
- [ ] Views should not perform any application logic. We can replace `User.find(post.user_id).name` with just `post.user.name`

Reply:
solved